website/source/guides/monitoring/prometheus-metrics.html.md

```diff
@@ -0,0 +1,573 @@
+    task "server" {
+      driver = "exec"
+      config {
+        command = "/local/demo"
+      }
+
+      artifact {
+        source = "https://s3-us-west-2.amazonaws.com/hashicorp-education/demo-binaries/prometheus-instrumentation"
+        destination = "local/demo"
+        mode = "file"
+      }
+
+      resources {
+        cpu = 500
+        memory = 256
+        network {
+          mbits = 10
+          port  "http"{}
+        }
+      }
+
+      service {
+        name = "webserver"
+        port = "http"
+
+        tags = [
+          "testweb",
+          "urlprefix-/webserver strip=/webserver",
+        ]
+
+        check {
+          type     = "http"
+          path     = "/"
+          interval = "2s"
+          timeout  = "2s"
+        }
+      }
+    }
+  }
+}
+```
+At this point, re-run your Prometheus job. After a few seconds, you will see the
+web server and Alertmanager appear in your list of targets.
+
+[![New Targets][new-targets]][new-targets]
+
+You should also be able to go to the `Alerts` section of the Prometheus web interface
+and see the alert that we have configured. No alerts are active because our web server
+is up and running.
+
+[![Alerts][alerts]][alerts]
+
+### Step 9: Stop the Web Server
+
+Run `nomad stop webserver` to stop our webserver. After a few seconds, you will see
+that we have an active alert in the `Alerts` section of the web interface.
+
+[![Active Alerts][active-alerts]][active-alerts]
+
+We can now go to the Alertmanager web interface to see that Alertmanager has received
+this alert as well. Since Alertmanager has been configured behind fabio, go to the IP address of any of your client nodes at port `9999` and use `/alertmanager` as the route. An example is shown below:
+
+-> < client node IP >:9999/alertmanager
+
+You should see that Alertmanager has received the alert.
+
+[![Alertmanager Web UI][alertmanager-webui]][alertmanager-webui]
+
+## Next Steps
+
+Read more about Prometheus [Alertmanager][alertmanager] and how to configure it
+to send out notifications to a [receiver][receivers] of your choice.
+
+[active-alerts]: /assets/images/active-alert.png
+[alerts]: /assets/images/alerts.png
+[alerting]: https://prometheus.io/docs/alerting/overview/
+[alertingrules]: https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/
+[alertmanager]: https://prometheus.io/docs/alerting/alertmanager/
+[alertmanager-webui]: /assets/images/alertmanager-webui.png
+[configuring prometheus]: https://prometheus.io/docs/introduction/first_steps/#configuring-prometheus
+[consul_sd_config]: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#%3Cconsul_sd_config%3E
+[env]: /docs/runtime/environment.html
+[fabio]: https://fabiolb.net/
+[fabio-lb]: http://example.com
+[new-targets]: /assets/images/new-targets.png
+[prometheus-targets]: /assets/images/prometheus-targets.png
+[running-jobs]: /assets/images/running-jobs.png
+[telemetry]: /docs/agent/telemetry.html
```

Comment:
This Telemetry reference page was moved to `guides/operations/monitoring/telemetry.html`. I suggest moving the page itself to `docs/telemetry.html` at this point with the main navigation link directly under `Variable Interpolation`. You will also need to change the link here.